program/rust/src/tests/pyth_simulator.rs

```diff
@@ -78,11 +78,7 @@ lazy_static::lazy_static! {
 
     // Detect features and pass them onto cargo-build-bpf.
     // IMPORTANT: All features of this crate must have gates added to this vector.
-    let features: Vec<&str> = vec![
-    #[cfg(feature = "pythnet")]
-    "pythnet",
-
-    ];
+    let features: Vec<&str> = vec!["pythnet"];
```

Comment:
we won't need it anymore right?

Reply:
Yes you can get rid of the whole `lazy_static`